### ISSUE TITLE
suggest proton as an experimental feature for users who don't have proton for all other titles enabled when they wanna play a windows only game

### ISSUE THREAD

**Issue by Weirdo1312** [ValveSoftware/steam-for-linux#7234]
I still see many people who have steam installed on their Linux box, but they're not aware of proton since it's not enabled by default
it would be good to let them know that such feature exists

**Comment by NikoBellicRU** [comment]
Yeah i agreed that it should be more informative for new users, soon as you install steam the game just apears as a windows game and you can't click, would be better to offer the option to enable it when clicking install and warning the user that he is using a compatibility tool.

**Comment by defiling9046** [comment]
Is this ever going to happen?

**Comment by sull324** [comment]
yes,should be about time.

**Comment by defiling9046** [comment]
> I don't understand why this request is the one being considered instead of this one : [#11707](https://github.com/ValveSoftware/steam-for-linux/issues/11707)
> 
> Valve says it can't be enabled due to some games running on proton being unofficial, but why is it on by default on the steamdeck then ?, why is the steamdeck treated differently to other linux distros ?
> 
> While i think this option would be better than nothing, i see no reason on why it should be different than the steamdeck.

Whilst I agree with you I think this is better than nothing.

I feel that as long as this is implemented as some sort of pop up that shows whenever a Windows only game is clicked on and gives the option for the user to enable Proton by default for Non-Windows games in that same pop up this approach isn’t the worst.

@kisak-valve Is there anything you think could be improved on my pop-up implementation. I was thinking the pop up could be similar to how the “processing Vulcan shaders” works but with the buttons at the bottom of the pop-up giving the user either the option to “ignore and exit game” or “enable proton” and this pop-up would happen every time a Linux user tries to play a Windows only game so it cannot be easily dismissed or accidentally clicked off by mistake never to be seen again.

**Comment by Weirdo1312** [comment]
> Is this ever going to happen?

who cares anyway

**Comment by RokeJulianLockhart** [comment]
> https://github.com/ValveSoftware/steam-for-linux/issues/7234#event-16100284444

@Weirdo1312, if you're closing it because you don't want it tracked anymore, close as invalid, not completed. Don't make the triage assignee's life harder.

**Comment by Weirdo1312** [comment]
> > [#7234 (comment)](https://github.com/ValveSoftware/steam-for-linux/issues/7234#event-16100284444)
> 
> [@Weirdo1312](https://github.com/Weirdo1312), if you're closing it because you don't want it tracked anymore, close as invalid, not completed. Don't make the triage assignee's life harder.
> 
> > [#7234 (comment)](https://github.com/ValveSoftware/steam-for-linux/issues/7234#issuecomment-2620205392)
> 
> Does anyone want to refile this?

that's a valid argument

**Comment by defiling9046** [comment]
Would love an update as I don't want this to be forgotten. @kisak-valve in my last message I left a suggestion of how you could implement it.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/ValveSoftware/steam-for-linux/issues/7234#issuecomment-2667474863

@defiling9046, it's not forgotten, because it's tracked in an open issue. Consequently, Kisak has almost definitely triaged it by now.

Regardless, the difficulty in implementing a feature (that isn't intended to be standardised across multiple implementations) is the implementation phase. Valve is capable of ascertaining how to design the GUI.

**Comment by Newbytee** [comment]
It seems like it's happening now: https://www.gamingonlinux.com/2025/06/steam-beta-finally-enables-proton-on-linux-fully-making-linux-gaming-simpler/

**Comment by GuardianLiarus** [comment]
> It seems like it's happening now: https://www.gamingonlinux.com/2025/06/steam-beta-finally-enables-proton-on-linux-fully-making-linux-gaming-simpler/

Can indeed confirm, I'm using the steam Beta client on my laptop and it's on by default!

**Comment by 06kellyjac** [comment]
Can this now be closed as resolved @kisak-valve ? 🙂

**Comment by kisak-valve** [comment]
Closing per the last handful of comments.
